sdk/healthinsights/azure-healthinsights-cancerprofiling/samples/sample_infer_cancer_profiling.py

```diff
@@ -0,0 +1,199 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import asyncio
+import os
+import datetime
+
+from azure.core.credentials import AzureKeyCredential
+from azure.healthinsights.cancerprofiling.models import *  # type: ignore
+from azure.healthinsights.cancerprofiling.aio import CancerProfilingClient
+
+"""
+FILE: sample_infer_cancer_profiling.py
+
+DESCRIPTION:
+    Infer key cancer attributes such as tumor site, histology, clinical stage TNM categories and pathologic stage TNM 
+    categories from a patient's unstructured clinical documents.
+    
+    OncoPhenotype model enables cancer registrars and clinical researchers to infer key cancer attributes from 
+    unstructured clinical documents along with evidence relevant to those attributes. This model can help reduce the 
+    manual time spent combing through large amounts of patient documentation. 
+
+
+USAGE:
+    python sample_infer_cancer_profiling.py
+
+    Set the environment variables with your own values before running the sample:
+    1) HEALTHINSIGHTS_KEY - your source from Health Insights API key.
+    2) HEALTHINSIGHTS_ENDPOINT - the endpoint to your source Health Insights resource.
+"""
+
+
+class HealthInsightsSamples:
+    async def infer_cancer_profiling(self):
```

Comment:
this is async, we only do sync in `samples/` and have async samples in `samples/async_samples`, and we should have both a sync and async version of `sample_infer_cancer_profiling.py`, you can look at the sample structure in text analytics: https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/textanalytics/azure-ai-textanalytics/samples

Reply:
done